<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能驾驶系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>881</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>791</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>871</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QLabel" name="testLabel">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>选择视频文件:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblStatus">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>15</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBrowseCam">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="sideBar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>191</width>
      <height>621</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnCloseCam">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>280</y>
       <width>191</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCam">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>180</y>
       <width>191</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>打开视频文件</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnUser">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>50</y>
       <width>191</width>
       <height>93</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;
    min-height: 80px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

</string>
     </property>
     <property name="text">
      <string>智能驾驶系统</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnSettings">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>370</y>
       <width>191</width>
       <height>93</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>86</width>
       <height>90</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;
    min-height: 80px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97);
	background-color: rgb(246, 246, 246);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

</string>
     </property>
     <property name="text">
      <string>系统设置</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSwitch">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>229</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>切换模式</string>
     </property>
    </widget>
    <zorder>btnSwitch</zorder>
    <zorder>btnCloseCam</zorder>
    <zorder>btnOpenCam</zorder>
    <zorder>btnUser</zorder>
    <zorder>btnSettings</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
